net-analyzer/prelude-lml/prelude-lml-4.0.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Tests are doubly broken in this package:

```
make[1]: Entering directory '/tmp/portage/net-analyzer/prelude-lml-4.0.0/work/prelude-lml-4.0.0/tests'
./loggrep.py ../plugins/pcre/ruleset/*.rules | ../src/prelude-lml --quiet --dry-run --metadata=nowrite,head --batch-mode --no-resolve --pcre --dump-unmatched --config ./prelude-lml.conf 2>&1 | /bin/grep -Fvf ./ignored
  File "./loggrep.py", line 29
    print i[:-1]
          ^
SyntaxError: Missing parentheses in call to 'print'
Invalid option -- "pcre" (0).
Invalid option -- "dump-unmatched" (0).
./prelude-lml.conf:52: invalid section : "Pcre".
./prelude-lml.conf:53: invalid option "ruleset" in "global" section.
03 Sep 12:01:08 (process:14720) ERROR: couldn't open config file /etc/prelude-lml/plugins.rules. (regex.c:168 get_regex_table)
03 Sep 12:01:08 (process:14720) WARNING: error while setting option 'file'.
make[1]: Leaving directory '/tmp/portage/net-analyzer/prelude-lml-4.0.0/work/prelude-lml-4.0.0/tests'
```

Firstly, because you don't set up Python for them while they clearly need Python 2. Secondly, because the make target ignores the resulting failure.

Reply:
Tests are not enable in this ebuid because for now, its bugguy but upstream will works on this